apps/emqx_conf/src/emqx_conf_cli.erl

```diff
@@ -118,6 +130,34 @@ admins(["fast_forward", Node0, ToTnxId]) ->
 admins(_) ->
     emqx_ctl:usage(usage_sync()).
 
+fix_lagging_with_raw(ToTnxId, Node, Keys) ->
+    Confs = lists:foldl(
+        fun(Key, Acc) ->
+            KeyRaw = atom_to_binary(Key),
+            Acc#{KeyRaw => emqx_conf_proto_v4:get_raw_config(Node, [Key])}
+        end,
+        #{},
+        Keys
+    ),
+    case mark_fix_begin(Node, ToTnxId) of
+        ok ->
+            case load_config_from_raw(Confs, #{mode => replace}) of
+                ok -> waiting_for_fix_finish();
+                Error0 -> Error0
+            end;
+        {error, Reason} ->
+            emqx_ctl:warning("mark fix begin failed: ~s~n", [Reason])
+    end.
+
+mark_fix_begin(Node, TnxId) ->
+    {atomic, Status} = emqx_cluster_rpc:status(),
+    MFA = {?MODULE, mark_fix_log, [Status]},
+    emqx_cluster_rpc:update_mfa(Node, MFA, TnxId).
+
+mark_fix_log(Status, Opts) ->
+    ?SLOG(warning, #{msg => cluster_fix_log, status => Status, opts => Opts}),
```

Comment:
we'll need to avoid dumping everything to logs ?
```suggestion
    ?SLOG(warning, #{msg => cluster_config_sync_triggered, status => format_status(Status), opts => Opts}),
```